modules/mobile/app/services/mobile/v0/lighthouse_claims/claims_index_interface.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'lighthouse/facilities/client'
+require 'lighthouse/benefits_claims/service'
+require_relative '../claims/proxy'
+
+module Mobile
+  module V0
+    module LighthouseClaims
+      class ClaimsIndexInterface
+        CLAIMS_NOT_AUTHORIZED_MESSAGE = 'Forbidden: User is not authorized for claims'
+        APPEALS_NOT_AUTHORIZED_MESSAGE = 'Forbidden: User is not authorized for appeals'
+        def initialize(user)
+          @current_user = user
+        end
+
+        def get_accessible_claims_appeals(use_cache)
+          if claims_access? && appeals_access?
+            get_claims_and_appeals(use_cache)
+          elsif claims_access?
+            get_claims(use_cache)
+          elsif appeals_access?
+            get_appeals(use_cache)
+          else
+            raise Pundit::NotAuthorizedError
+          end
+        end
+
+        def try_cache(data, errors)
```

Comment:
Given that both uses of this class fetch get_accessible_claims_appeals, which returns data and errors, which we then feed into this method, I think you could simplify the process slightly by using this method in get_accessible_claims_appeals.